src/NHibernate.ProxyGenerators/Default/DefaultProxyGenerator.cs

```diff
@@ -89,6 +89,12 @@ protected virtual CompilerResults CompileStaticProxyFactory(Configuration nhiber
 			foreach (var cls in nhibernateConfiguration.ClassMappings)
 			{
 				references.Add(cls.MappedClass.Assembly);
+				foreach (var mapReferenceAssemblyName in cls.MappedClass.Assembly.GetReferencedAssemblies())

```

Comment:
Sure, though I won't be able to unless it depends on FluentNHibernate.  I'll see when I have time to do that.